src/generator/templates/api/lib/mbuild.mako

```diff
@@ -703,17 +698,6 @@ else {
             request_value_reader.seek(io::SeekFrom::Start(0)).unwrap();
             % endif
             let mut req_result = {
-            % if resumable_media_param:
```

Comment:
Would this not be missing somewhere?

Reply:
I was planning to ask you about this. It was getting in the way, but I could not find how it was actually used. It looks like dead code. The way I see it is that `should_ask_dlg_for_url ` is always false. Or what is going on here?

Reply:
I was planning to ask you about this. The way I see it `should_ask_dlg_for_url` is always false. So that looked like a piece of dead code. Or what is going on here?

Reply:
Right! I checked and the variable was false from the very beginning, so it could never have worked anyway.

Also something I just noticed is the 'assignment as side-effect of if-condition' dark magic that is performed here, and it's incredible that I was the author of this 😅.

In any case, I agree that under the circumstances, it should definitely be removed.